thoth/common/openshift.py

```diff
@@ -1520,6 +1521,7 @@ def schedule_mi_workflow(
             "REPOSITORY": repository,
             "ENTITIES": entities,
             "KNOWLEDGE_PATH": knowledge_path,
+            "MI_THOTH": "true" if mi_thoth else "false",
```

Comment:
why this booleans are strings?

Reply:
it's a specified parameter that will be used as envvar for setting the cli option `-t`, and the envvar are always strings (e.g. see https://github.com/thoth-station/common/blob/5b96cd3c2ddf4856426f69f87421215643f3f4cc/thoth/common/openshift.py#L1368 - setting integer, however in string form, because of envvar)
I don't know whether `click` supports assigning "1" or "0" as a parameter, but I know that it supports strings "false" and "true" when it comes to setting flags by envvars